### PR TITLE
JTAG cable check task for no-OS

### PR DESCRIPTION
**Pull request by tagoylo**

### CODE REVIEW

File: nebula/manager.py [hunk 1]
```diff
@@ -54,7 +54,9 @@ def __init__(
                     if self.jtag_use:
                         try:
                             self.jtag = jtag(
-                                yamlfilename=configfilename, board_name=board_name
+                                yamlfilename=configfilename,
+                                board_name=board_name,
+                                vivado_version=extras,
```
Comment (jpineda3) [inline nebula/manager.py:59]: Might as well add vivado version parameter.  @tfcollins Any reason for adding extras here?

File: nebula/tasks.py [hunk 2]
```diff
@@ -380,6 +380,24 @@ def recovery_device_manager(
         m.board_reboot_auto_folder(folder, sdcard, design_name=board_name, recover=True)
 
 
+@task(
+    help={
+        "yamlfilename": "Path to yaml config file. Default: /etc/default/nebula",
+        "board_name": "Name of DUT design (Ex: zynq-zc706-adv7511-fmcdaq2). Require for multi-device config files",
```
Comment (jpineda3) [inline nebula/tasks.py:386]: Update this to include vivado version.

Comment (tagoylo) [reply nebula/tasks.py:387]: Addressed through commit [268c93d](https://github.com/sdgtt/nebula/pull/78/commits/268c93da8c19a54a651d46defa5ee04311a02640).

File: nebula/manager.py (hunk 1 above)
Comment (tagoylo) [reply nebula/manager.py]: Addressed through commit [268c93d](https://github.com/sdgtt/nebula/pull/78/commits/268c93da8c19a54a651d46defa5ee04311a02640).

**Review by kimpaller — APPROVED** [review]

**Pull request merged**